debian/mariadb-plugin-connect.install

```diff
@@ -1,2 +1,4 @@
 etc/mysql/conf.d/connect.cnf  etc/mysql/mariadb.conf.d
+usr/lib/mysql/plugin/JavaWrappers.jar
+usr/lib/mysql/plugin/JdbcInterface.jar
```

Comment:
This is not build without openjdk-sdk-headless.

Reply:
What should be about Mongo support which is using  `usr/share/mysql/Mongo2.jar`  and `usr/share/mysql/Mongo3.jar` ?

Reply:
What should be about Mongo support which is using  `usr/share/mysql/Mongo2.jar`  and `usr/share/mysql/Mongo3.jar` ?

Reply:
I think that i worth of another PR after this one is in.

Reply:
@ottok do you think we can revert to the d7321893d8c50071632a102e17a7869da9cb03a5 and solve the problem of 3c922d6defcfa6be819fe33794abcf507bb70c25 on buildbot side ?

Reply:
What do you mean _3c922d6 on buildbot side_? Having binaries in source control is plain wrong. Making it possible with some hacks is not the solution, but to fix the build so that the whole project builds from source code to binary. That is almost already done if we just figure out what is the correct dependency.